vite.config.ts

```diff
@@ -1,16 +1,17 @@
-import { defineConfig } from "vite";
+import { defineConfig, loadEnv } from "vite";
 import react from "@vitejs/plugin-react-swc";
 import tsconfigPaths from "vite-tsconfig-paths";
-import dotenv from "dotenv";
-
-dotenv.config();
 
 // https://vitejs.dev/config/
-export default defineConfig({
-  plugins: [react(), tsconfigPaths()],
-  server: {
-    proxy: {
-      "/api": process.env.API_SERVER_URL,
+export default defineConfig(({ mode }) => {
+  const TmpEnv = loadEnv(mode, process.cwd());
```

Comment:
TmpEnv는 상수이므로 lowerCamelCase를 사용하시면 됩니다!
```suggestion
  const env = loadEnv(mode, process.cwd());
```

Reply:
Rgr